### DISCUSSION TITLE
Comments in debugger scripts

### DISCUSSION THREAD

**Discussion by jotego** [mamedev/discussions#34] [Q&A] [answered]
Is there a way to add a comment in a debugger script? 

I have tried the usual suspects like `# // /* */ --` but none of these work. The documentation doesn't seem to mention a way to add a comment. I know comments do not make much sense in the interactive use case but MAME allows for debugger script files too and comments make sense there.

**Comment by pmackinlay** [comment, marked as answer]
According to [this](https://github.com/mamedev/mame/blob/master/src/emu/debug/debugcon.cpp#L511) you should be able to use `//`, but probably only in a script and not interactively?

**Reply by jotego** [reply]
I see! It's only for files then, fair enough :-)

**Comment by angelosa** [comment]
`help comments` in debugger gives you all you need to know. CTRL+N changes display of raw opcodes to comments, cfr. Options if you need to switch back. 
![debugger_view](https://github.com/mamedev/discussions/assets/9251271/b8d39d67-372c-4ed6-aedc-48fb4a3b3e10)

**Reply by cuavas** [reply]
That’s not what @jotego is asking for.  He’s asking how to include comments in the script, not how to add annotations to the disassembly.

**Reply by angelosa** [reply]
Fair enough, got fooled by the `/*` `//` in OP comment.
